.github/workflows/build-msi-package.yml

```diff
@@ -0,0 +1,32 @@
+name: "Build MSI Package"
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+    build-msi-package:
+      runs-on: windows-latest
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v4
+
+        - name: Build C++ NuGet Packages
```

Comment:
Also slice compilers and services right?

Reply:
Yes. The NuGet packages include all C++ binaries.